superset/connectors/sqla/models.py

```diff
@@ -795,6 +795,10 @@ def get_sqla_query(  # sqla
                     'order_desc': True,
                 }
                 result = self.query(subquery_obj)
+
+                if result.status == utils.QueryStatus.FAILED:
+                    raise Exception(result.error_message)
```

Comment:
Let's create a custom exception for this, something like `SupersetSubQueryError`